ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/ccmv2/CcmV2ManagementClient.java

```diff
@@ -45,13 +47,15 @@ public InvertingProxy awaitReadyInvertingProxyForAccount(String requestId, Strin
                 });
     }
 
-    public InvertingProxyAgent registerInvertingProxyAgent(String requestId, String accountId, String domainName, String keyId) {
+    public InvertingProxyAgent registerInvertingProxyAgent(String requestId, String accountId, Optional<String> environmentCrnOpt,
+        String domainName, String keyId) {
         return getRetryTemplate().execute(
                 retryContext -> {
-                    LOGGER.debug("Registering Agent for accountId '{}', domainName '{}'", accountId, domainName);
-                    InvertingProxyAgent invertingProxyAgent = grpcCcmV2Client.registerAgent(requestId, accountId, domainName, keyId,
+                    LOGGER.debug("Registering Agent for accountId '{}', environmentCrnOpt: {}, domainName '{}'", accountId, environmentCrnOpt, domainName);
+                    InvertingProxyAgent invertingProxyAgent = grpcCcmV2Client.registerAgent(requestId, accountId, environmentCrnOpt, domainName, keyId,
                             ThreadBasedUserCrnProvider.getUserCrn());
-                    LOGGER.debug("Registered Agent for accountId '{}', domainName '{}', invertingProxyAgent '{}'", accountId, domainName, invertingProxyAgent);
+                    LOGGER.debug("Registered Agent for accountId '{}', environmentCrnOpt: {}, domainName '{}', invertingProxyAgent '{}'",
```

Comment:
nit: missing `''` in  `environmentCrnOpt: {}`

Reply:
done.